gdb-server/src/bin.rs

```diff
@@ -30,6 +30,15 @@ struct Opt {
         help = "Use this flag to override the default GDB connection string (localhost:1337)."
     )]
     gdb_connection_string: Option<String>,
+    #[structopt(name = "list", long = "list", help = "list available debug probes")]
+    list: bool,
```

Comment:
Could you name this `list-probes` as for most people it might suggest the listing of chips?